lib/puppet_x/puppetlabs/registry/provider_base.rb

```diff
@@ -28,10 +28,9 @@ def self.define_ffi
     #   _Inout_opt_  LPDWORD lpcbData
     # );
     ffi_lib :advapi32
-    attach_function :RegQueryValueExA,
+    attach_function :RegQueryValueExW,
       [:handle, :pointer, :pointer, :pointer, :pointer, :pointer], :win32_long
 
-    # http://msdn.microsoft.com/en-us/library/windows/desktop/ms724847(v=vs.85).aspx
```

Comment:
Was this intentional? Always appreciate links - https://msdn.microsoft.com/en-us/library/windows/desktop/ms724847.aspx